supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -324,12 +324,22 @@ There is no functional difference between the first server that was installed an
 
 [[ingress]]
 ==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The `Ingress` resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
```

Comment:
@bergerhoffer I added markup for mentionings of the `Ingress` resource. I also added some context to the *Incorrect forms* description. Wdyt? Does this clarify the ambiguity a little further? Any other things that are a little unclear still? 🤔

Reply:
@skrthomas Sorry for the late reply. The markup around `Ingress` makes sense to me. But the entry for capitalized "Ingress" still doesn't.

Per pretty much every other OCP object we have, we have the actual API object marked up when referencedd (`Pod`), but then when we speak of it generically, it's lowercase (pod). So this uppercase "Ingress" entry doesn't make sense to me. It doesn't seem like there should be a reason that "Ingress" is used but not marked up.

Or... is that the problem? Is there really no legitimate use of when "ingress"/"Ingress" should be used as a generic reference? Should it either be the `Ingress` API object (marked up), or otherwise it needs to be "ingress" as an adjective, in front of a noun?

Reply:
@bergerhoffer 
> It doesn't seem like there should be a reason that "Ingress" is used but not marked up.

The reason ingress could be used but not marked up would be when it is used to describe the direction of traffic flow, which is why I was thinking it could be its own entry in the Glossary. But I guess, further thinking on this, I'm wondering if we can keep a single "ingress" definition, and clarify that the Ingress resource exposes HTTP and HTTPS routes from outside the cluster to _services within the cluster_, basically routing incoming traffic to different cluster applications. 

Then we can make it clearer that you can refer to the `Ingress` object, which must be marked up, or you can describe the general directional flow of incoming traffic, which does not need to be marked up and can be lowercase. 

> Is there really no legitimate use of when "ingress"/"Ingress" should be used as a generic reference? Should it either be the Ingress API object (marked up), or otherwise it needs to be "ingress" as an adjective, in front of a noun?

In my experience writing about Networking, I've only ever used it to either reference the `Ingress Controller` resource or to describe directional flow of traffic. Maybe these are the two use cases similar to `Pod` and pod.